examples/cookbook/design/drawer/lib/drawer.dart

```diff
@@ -11,6 +12,7 @@ void drawerStart() {
 void drawerEmpty() {
   // #docregion DrawerEmpty
   Scaffold(
+    appBar: AppBar(),
```

Comment:
Add a `title` here as well. I'd change it to something like 'AppBar with hambuger button'.

Reply:
Same changes here.